app/views/media/show.html.erb

```diff
@@ -1,9 +1,22 @@
 <h2><center><%= @media.title %></center></h2>
 
+
+
 <div class="row">
   <% if @media.poster %>
     <div class='poster col-md-3'>
       <%= image_tag(@media.poster, height:'100%', width:'100%') %>
+        <div id="add_to_list">
+          <% if @user %>
+            <div id="current_list">
+              Current List:
+              <%= @media.user_lists %>
+            </div>
+            <%= render partial: 'shared/drop_down_lists', locals: { user_id: @user.id, media_id: @media.id, button_text: "Add to List"}  %>
```

Comment:
I wonder how difficult it would be for the button text to default to current list if its already on a list?